website/docs/feature-flag-tutorials/use-cases/a-b-testing.md

```diff
@@ -29,84 +31,123 @@ The feature flag we are creating is considered an ‘Experimentation’ flag typ
--   **weight**: The weight is the likelihood of any one user getting this specific variant. See the weights section for more info.
--   **value**
--   **(optional) payload**: A variant can also have an associated payload. Use this to deliver more data or context. See the payload section for more details.
+-   **a name**: This must be unique among the strategy's variants. When working with a feature with variants in a client, you will typically use the variant's name to find out which variant it is.
+-   **a weight**: The weight is the likelihood of any one user getting this specific variant. See the [weights section](/reference/strategy-variants#variant-weight) for more info.
+-   **an optional payload**: A variant can also have an associated payload. Use this to deliver more data or context. See the [payload section](/reference/strategy-variants#variant-payload) for more details.
+-   **a value**
+
+While teams may have different goals for measuring performance, Unleash enables you to configure strategy variants within your application/service and the platform.
+
+![You can review the anatomy of an Unleash strategy variants in relation to other Unleash features.](/img/anatomy-of-unleash-variants.png)
 
-While teams may have different goals for measuring performance, Unleash enables you to configure a strategy for the feature variants within your application/service and the platform.
+In your gradual rollout form, there is a ‘Variants’ section. Click on the ‘Add variant’ button. A form will appear where you can provide a unique name for the variant. For the purpose of this tutorial, we’ve created 2 variants shown in the image below: `variantA` and `variantB`. In a real world use case, we recommend more specific names to be comprehensible and relevant to the versions of the feature you’re referencing. Create additional variants if you need to test more versions.
 
-## A/B Testing with Enterprise Security Automation
+Next, decide on the percentage of users that will be targeted for each variant, which is known as the _variant weight_. By default, 50% of users will be targeted between 2 variants. For example, 50% users within the 35% of users targeted from the rollout percentage you defined earlier would experience `variantA`. Switch on the ‘Custom percentage’ toggle to change the default of the variant weights.
 
-For large-scale organizations, managing feature flags across many teams can be complex and challenging. Unleash was architected for your feature flag management to be scalable and traceable for enterprises, which boosts overall internal security posture while delivering software efficiently.
+![You can configure multiple strategy variants for A/B testing within the gradual rollout form.](/img/use-case-variants.png)
 
-After you have implemented an A/B test, we recommend managing it by:
+To learn more, read our documentation on [strategy variants](/reference/strategy-variants).
```

Comment:
Docs linked at first mention of strategy variants already, so I think we can skip this sentence to keep the tutorial focused. 
```suggestion
```